tests/basic/pure_test.go

```diff
@@ -4238,6 +4238,248 @@ var _ = Describe("{CreateNewPoolsWhenFadaFbdaVolumeCreationInProgress}", func()
+				2. Create 10 PVC each with respective storage class parallely
+				3. Validate if PVC are created and bounded
+				4. Validate if corresponding portworx volumes are created in FA backend
+				5. Validate if corresponding portworx volumes are created in FB backend
+				6. Delete the pvc and volume and check if volumes got deleted in backend as well
+	*/
+	JustBeforeEach(func() {
+		StartTorpedoTest("CreateAndValidatePVCWithIopsAndBandwidth",
+			"Create PVCs with updated MaxBandwidth / Max IOPS ( update the storage class )",
+			nil, 0)
+	})
+	itLog := "CreateAndValidatePVCWithIopsAndBandwidth"
+	It(itLog, func() {
+		log.InfoD(itLog)
+		numberOfPvc := 10
+		var k8sCore = core.Instance()
+		var wg sync.WaitGroup
+		var max_bandwidth uint64
+		var max_iops uint64
+
+		//Declaring SC name, namespaces and pvc prefixes and lists which are required for collection of PVC And Volume Names
+		baseScName := "base-portworx-volume-sc"
+		fadaScName := "fada-volume-sc"
+		fbdaScName := "fbda-volume-sc"
+		BaseAppNameSpace := "base-app-namespace"
+		FadaAppNameSpace := "fada-app-namespace"
+		FbdaAppNameSpace := "fbda-app-namespace"
+		max_iops = 1000
+		max_bandwidth = 1
+		//Creating Two lists to collect the volume names of both FA and FB created volumes
+		listofFadaPvc := make([]string, 0)
+		listofFbdaPvc := make([]string, 0)
+		//Creating maps with namespaces as key and list of pvc names as values
+		namespaces := []string{FadaAppNameSpace, FbdaAppNameSpace, BaseAppNameSpace}
+		namespacePVCMap := make(map[string][]string)
+		for _, ns := range namespaces {
+			namespacePVCMap[ns] = []string{}
+		}
+		//Creating map with storage class as key and namespace as value
+		scNamespaceMap := map[string]string{
+			baseScName: BaseAppNameSpace,
+			fadaScName: FadaAppNameSpace,
+			fbdaScName: FbdaAppNameSpace,
+		}
+
+		//Get The Details of Existing FA AND FB in the cluster
+		flashArrays, err := GetFADetailsUsed()
+		log.FailOnError(err, "Failed to get FA details from pure.json in the cluster")
+		flashBlades, err := GetFBDetailsFromCluster()
+		log.FailOnError(err, "Failed to get FB details from pure.json in the cluster")
+
+		stepLog := "Create storage class with max iops and max bandwidth for Normal Portworx Volumes , FADA and FBDA Pvc Deployment"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			BaseParams := make(map[string]string)
+			BaseParams["repl"] = "1"
+			BaseParams["max_iops"] = "1000"
+			BaseParams["max_bandwidth"] = "1G"
+			reclaimPolicyDelete := v1.PersistentVolumeReclaimDelete
+			bindMode := storageApi.VolumeBindingImmediate
+			// create storage class for base volumes
+			_, err := CreatePortworxStorageClass(baseScName, reclaimPolicyDelete, bindMode, BaseParams)
+			log.FailOnError(err, "Failed to create base storage class")
+			log.InfoD("Storage class [%s] for Basic is created", baseScName)
+
+			faParams := make(map[string]string)
+			faParams["repl"] = "1"
```

Comment:
[nit] Unless we're testing that setting "repl" fails, we should not set "repl" for an FA/FB storage class.